Create hero section for multimedia pages

gabrieltanchen opened the pull request:
#### Description of changes
resolves CIGIHub/cigi-tickets#251

This includes the javascript for the sticky video player when scrolling.

Regular:
![Screenshot from 2020-12-21 11-06-32](https://user-images.githubusercontent.com/10100465/102796818-d3011000-437c-11eb-99e9-59473e57e01d.png)

Sticky:
![Screenshot from 2020-12-21 10-59-44](https://user-images.githubusercontent.com/10100465/102796851-dac0b480-437c-11eb-879c-79f8442f050a.png)

johnbxu commented:
Between 768px and 991px There is extra margin on the top of the hero due to the header shrinking around 991px:

![image](https://user-images.githubusercontent.com/40705848/102800510-faa6a700-4381-11eb-9270-238ca118255f.png)

gabrieltanchen commented:
@johnbxu can you double check the margin? It may have gotten fixed when I merged... not sure what happened there.

I also cleaned up some of the code, moved some of the CSS and JS to helper files as I'll need them for Big Tech.

johnbxu commented:
I think there must be some caching rules in chrome that's making this happen. I'll start manually clearing caches when I swap branches.
